src/mouse.c

```diff
@@ -183,11 +183,10 @@ void switch_virtual_desktop(device_t *state, output_t *output, int new_index, in
           )___(          )___(     |     )___(          )___(          )___(
 */
 void check_screen_switch(const mouse_values_t *values, device_t *state) {
-    int new_x        = state->pointer_x + values->move_x;
     output_t *output = &state->config.output[state->active_output];
 
-    bool jump_left  = new_x < MIN_SCREEN_COORD - state->config.jump_treshold;
-    bool jump_right = new_x > MAX_SCREEN_COORD + state->config.jump_treshold;
+    bool jump_left  = state->pointer_x <= MIN_SCREEN_COORD + state->config.jump_treshold;
```

Comment:
If you move the mouse to MIN_SCREEN_COORD on one device, you'd get transferred to MAX_SCREEN_COORD on the other and immediately get transferred back?

Also, this effectively leaves you without a way to ever click on the first and last pixel columns on their respective screens?

Reply:
Hmn, okay… but can you even go below MIN_SCREEN_COORD (0) or further then MAX_SCREEN_COORD?

Reply:
Physically - no, but you could request it and that in theory should trigger screen switch if there is a screen to switch to, or put you at the very edge if there isn't.

New_x is a bad variable name, in this case it's desired_x_coordinate, it doesn't get assigned to anything but is merely used to determine if you tried to leave a screen.